### PR TITLE
Active hover adjustment

### PR DESCRIPTION
**Pull request by jdtoombs**
Noticed a few places where the rows were still all blue, this fixes that.

### CODE REVIEW

File: libs/npm/core/src/components/grid-table/GridTable.tsx [hunk 1]
```diff
@@ -225,7 +225,7 @@ export const GridTable = <T extends object>({
             prepareRow(row);
             return (
               <div
-                className={activeId && activeId === (row.original as any).id ? 'active' : ''}
+                className={!!activeId && activeId === (row.original as any).id ? 'active' : ''}
```
Comment (Fosol) [inline libs/npm/core/src/components/grid-table/GridTable.tsx:228]: Why was this needed?

**Review by Fosol — APPROVED** [review]

File: libs/npm/core/src/components/grid-table/GridTable.tsx (hunk 1 above)
Comment (jdtoombs) [reply libs/npm/core/src/components/grid-table/GridTable.tsx:228]: For some reason on the time log table when no prop was provided it was still triggering.. but it had different behaviour on other pages (like ingestion)
